### PR TITLE
Fix crash from drawing zero length time graphs.

### PR DESCRIPTION
**Pull request by danielfenner**

### CODE REVIEW

File: src/OrbitGl/TimeGraph.cpp [hunk 1]
```diff
@@ -660,8 +660,11 @@ void TimeGraph::PrepareBatcherAndUpdatePrimitives(PickingMode picking_mode) {
   uint64_t min_tick = GetTickFromUs(min_time_us_);
   uint64_t max_tick = GetTickFromUs(max_time_us_);
 
-  CaptureViewElement::UpdatePrimitives(primitive_assembler_, text_renderer_static_, min_tick,
-                                       max_tick, picking_mode);
+  // Only draw something if the time interval is non-empty.
```
Comment (karupayun) [inline src/OrbitGl/TimeGraph.cpp:663]: What about?
`  // Only update the primitives to draw if the time interval is non-empty.
`

File: src/OrbitGl/TimeGraph.cpp [hunk 2]
```diff
@@ -660,8 +660,11 @@ void TimeGraph::PrepareBatcherAndUpdatePrimitives(PickingMode picking_mode) {
...
+  if (min_tick != max_tick) {
```
Comment (karupayun) [inline src/OrbitGl/TimeGraph.cpp:664]: To be consistent I would require min_tick < max_tick

**Review by karupayun — APPROVED** [review]
LGTM! Thanks for this Daniel. 

I'm surprised about why this never crashed before.

File: src/OrbitGl/TimeGraph.cpp (hunk 1 above)
Comment (danielfenner) [reply src/OrbitGl/TimeGraph.cpp]: sg. done.

File: src/OrbitGl/TimeGraph.cpp (hunk 2 above)
Comment (danielfenner) [reply src/OrbitGl/TimeGraph.cpp]: done

**Pull request merged**